ArduCopter/GCS_Copter.cpp

```diff
@@ -10,7 +10,7 @@ uint8_t GCS_Copter::sysid_this_mav() const
 const char* GCS_Copter::frame_string() const
 {
     if (copter.motors == nullptr) {
-        return "motors not allocated";
+        return "MultiCopter";
```

Comment:
We actually enter a config error loop if motors is not allocated. So we might as well return no string and save the flash.

Reply:
> Does this mean a user could forget to set FRAME_CLASS?

no, we still give the FRAME_CLASS error, we just don't block the UI from knowing what flight modes are suitable (and other UI elements)

Reply:
we could remove these nullptr checks completely if @rmackay9 is happy to do that, certainly there are lots of other places that dereference motors without checking for nullptr. Up to Randy really